### ISSUE TITLE
Upgrade to 4.1.0

### ISSUE THREAD

**Issue by drfurtado** [rstudio/revealjs#84]
How do I upgrade to revealjs 4.1.0?

Much appreciated.

**Comment by ccamara** [comment]
I haven't tried before (to be honest, I do not know how to setup my environment to install and work with a customized/forked version of a library), but I'd try following this guideline https://revealjs.com/upgrading/ within the `inst/` folder, where I believe is where reveal.js resides (which is still in the 3.3.0.1 version from four years ago)

**Comment by SamGuay** [comment]
Hi @drfurtado! 

Thanks for the pointer, @ccamara, I think I have a fully-working up to date version of the package! I had to change a couple more things, but I haven't encountered any problem so far with the presentation I am currently designing right now. I also added a couple more plugins for fun. 

If you want to give it a try, you can have a look in [my repo](https://github.com/SamGuay/revealjs) or you can simply install my forked package using the devtools package:
```
# install.packages("devtools") # uncomment this line to install the devtools package.
devtools::install_github("samguay/revealjs")
```

Feel free to give any feedback, it is the first time I try to update a package on my own!

**Comment by SamGuay** [comment]
Apparently, I took a very long weekend :smile:. I can work on upgrading to 4.1.2 and open a PR if you welcome them @apreshill ?

Thanks

**Comment by apreshill** [comment]
Yes @SamGuay PR's most welcome! We probably will not be able to review until September though. Also I note there is a relevant discussion about updating in this PR: https://github.com/rstudio/revealjs/pull/59

**Comment by drfurtado** [comment]
`Update`
I can confirm that the problem is related to pandoc. The behavior described below started when I upgraded pandoc to 2.14.2. The problem was solved by reverting back to pandoc 2.14.0.

**Comment by cderv** [comment]
@drfurtado 

Can you open a new issue about this ? Specific to this Pandoc update, with the error message you get. We will look into it and try to adapt to Pandoc changes so that it works in both version. Also if you could try version 2.14.1 b/c they change some stuff in the template in this version. 

Thank you !
